types/gapi.client.classroom/tsconfig.json

```
@@ -14,11 +14,11 @@
         ],
         "types": [],
         "noEmit": true,
-        "forceConsistentCasingInFileNames": true,
+        "forceConsistentCasingInFileNames": true,        
```

trailing whitespace?

---

Oops, good catch, thanks!

```suggestion
        "forceConsistentCasingInFileNames": true,
```